.github/dependabot.yml

```diff
@@ -0,0 +1,11 @@
+# To get started with Dependabot version updates, we need to specify "devcontainers" as a
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
```

Comment:
```suggestion
# For more details, see https://containers.dev/guide/dependabot
```

@michaelbprice what do you think about the suggestion?

Reply:
incorporated the suggestion